vantiq-apps-development/1-day-workshop/docs/eng/Vantiq_resources_introduction.md

```diff
@@ -0,0 +1,176 @@
+}
+```  
+④ Store information that want to add, such as the location information of AI cameras.  
+``` sh
+{
+👉“camera_id”: "camera_001",
+“lan”: 35.6864604,
+“lon”: 139.7635769
+}
+```
+
+## <img src="../../imgs/Vantiq_resources_introduction/slide1.png" width=1.8%>3. Store and update the latest location information of security guards  
+
+<img src="../../imgs/Vantiq_resources_introduction/slide10.png" width=70%>  
+
+①
+```sh
+{
+"guard_id": "134678493_1",
+"type": "location",
+"lat": 35.6864604,
+"lon": 139.7635769,
+"time": 1631083251
+}
+```  
+② Add information using `Enrich` if E-mail address, phone number, etc. are required for notification.    
+③ Store only the latest one in *Type* by using `SaveToType` with `Upsert` setting in App.  
+
+## <img src="../../imgs/Vantiq_resources_introduction/slide1.png" width=1.8%>4. Comparing the location of the AI camera and security guards  
+
+<img src="../../imgs/Vantiq_resources_introduction/slide11.png" width=70%>  
+
+① Data to be passed to Procedure.  
+```sh
+{
+“camera_id”: "camera_001",
+👉“lan”: 35.6864604,
+👉“lon”: 139.7635769,
+“event_type”: "notice_001",
+“image”: "data:image/jpe...."
+}
+```
+② Call your own function which is programmed the logic for comparing positions, from App.    
+③ Specify the nearest security guard by comparing the location information contained in _the AI camera data which is being stream processing_ and the location information in _the security guard data stored in Type_.    
+④ Store the latest location information of the security guards.  
+
+
+## <img src="../../imgs/Vantiq_resources_introduction/slide1.png" width=1.8%>5. Notify the specified security guard  
+
+<img src="../../imgs/Vantiq_resources_introduction/slide12.png" width=70%>  
+
+
+① Transform data to meet the format required for notification. (Transformation)  
+② Source is used not only for receiving but also for sending.    
+③ E-mail, SMS, Slack, LINE, Teams, etc.    
+
+## <img src="../../imgs/Vantiq_resources_introduction/slide1.png" width=1.8%>Overview of Vantiq Application
+
+<img src="../../imgs/Vantiq_resources_introduction/slide13.png" width=70%>
+
+## <img src="../../imgs/Vantiq_resources_introduction/slide1.png" width=1.8%>Sample App  
+
+A sample App (consisting of App and Data Generator) can be found [here](../../conf/ai-camera-demo).    
+
+
+# Introduction to each Resource
+
+|Reource|Description|
+|:-----|:---|
+|Source|A client that sends and receives data. &nbsp; Sending and Receiving: MQTT, AMQP, Kafka &nbsp; Sending only: HTTP(S), E-mail, SMS|
+|Topic|An event bus that passes events between App in Vantiq, etc. It also functions as an endpoint of REST API.|
+|Type|Store the data. Resource which corresponds to the DB in Vantiq.|
+|App|The application itself which can be developed on GUI by combining prepared patterns and Procedure.|
+|Procedure|Resource which can be programmed by oneself. It is possible to use the original features by calling them in App and so on.|
```

Comment:
```suggestion
|Procedure| A resource in which the custom logic can be implemented. It can be called from various resources such as Apps and Procedures.|
```